plugins/module_utils/mso.py

```diff
@@ -879,7 +887,7 @@ def make_reference(self, data, reftype, schema_id, template):
             'templateName': template,
         }
 
-    def make_subnets(self, data, querier=True):
+    def make_subnets(self, data, querier=True, primary=True, virtual=True):
```

Comment:
I thought virtual and primary are not necessarily for all subnets in the list. This change is impact all current use of make_subnets.

Reply:
For example are those available under the subnet inside an EPG?

Reply:
Added is_bd_subnet option to cover all 3 parameters.